.chloggen/fixrecombinetimeout.yaml

```
@@ -0,0 +1,15 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/stanza
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: 1.Make the forceFlushTimeout compare with timeSinceFirstEntry not timeSinceLastEntry 2.set the period of ticker in transformer to be 1/5 forceFlushTimeout
```

I think this should be a more user friendly, but we can move this text into the subtext.

```suggestion
note: Fix issue where recombine operator would never flush.
```

---

done